backend/tests/v5/unit/models/users.test.js

```diff
@@ -498,6 +498,79 @@ const testGetUserByUsernameOrEmail = () => {
 	});
 };
 
+const formatNewUserData = (newUserData) => {
+	const formattedData = {
+		inactive: true,
+		firstName: newUserData.firstName,
+		lastName: newUserData.lastName,
+		email: newUserData.email,
+		mailListOptOut: !newUserData.mailListAgreed,
+		permissions: [{
+			user: newUserData.username,
+			permissions: ['teamspace_admin'],
+		}],
+		billing: {
+			billingInfo: {
+				firstName: newUserData.firstName,
+				lastName: newUserData.lastName,
+				countryCode: newUserData.countryCode,
+				company: newUserData.company,
+			},
+		},
+	};
+
+	formattedData.emailVerifyToken = {
+		token: newUserData.token,
+	};
+
+	return formattedData;
+};
+
+const testAddUser = () => {
+	describe('Add a new user', () => {
+		test('should add a new user', async () => {
+			const newUserData = {
+				username: 'newUser',
+				email: 'example@email.com',
+				password: 'newPassword',
+				firstName: 'new first name',
+				lastName: 'new last name',
+				mailListAgreed: true,
+				countryCode: 'GB',
+				company: '3D repo',
+			};
+
+			const authDB = { addUser: () => {} };
+			jest.spyOn(db, 'getAuthDB').mockImplementation(() => authDB);
+			const fn = jest.spyOn(authDB, 'addUser');
+
+			const expectedData = formatNewUserData(newUserData);
+			await User.addUser(newUserData);
+			expect(fn.mock.calls.length).toBe(1);
+			expect(fn.mock.calls[0][0]).toEqual(newUserData.username);
+			expect(fn.mock.calls[0][1]).toEqual(newUserData.password);
+			const userCustomData = fn.mock.calls[0][2];
+			expect(userCustomData).toHaveProperty('customData.createdAt');
+			expect(userCustomData).toHaveProperty('customData.emailVerifyToken.expiredAt');
+			expectedData.createdAt = userCustomData.customData.createdAt;
```

Comment:
these 3 lines can just be consolidated into 1 if you use the spread operator 

i.e.
````js
{ ... expectedData, createdAt: userCustomData.customData.createdAt /* ... */}
````